### PR TITLE
Resolved issue where New-CimSessionOption would fail in example text for DSC article

### PR DESCRIPTION
**Pull request by dcrreynolds**
# PR Summary
* Removed `$true` from switch parameters in New-CimSessionOption cmdlet in example code.
* Split command to multiple lines to remove scrolling in code box.
* Fixed minor indention inconsistency.

## PR Context

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [ ] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [x] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [ ] Preview content
- [ ] Version 7.1 content
- [ ] Version 7.0 content
- [ ] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/docs-conceptual/dsc/getting-started/lnxGettingStarted.md [hunk 1]
```diff
@@ -133,11 +133,18 @@ $Node = "ostc-dsc-01"
...
 #Ignore SSL certificate validation
-#$opt = New-CimSessionOption -UseSsl $true -SkipCACheck $true -SkipCNCheck $true -SkipRevocationCheck $true
+#$opt = New-CimSessionOption -UseSsl -SkipCACheck -SkipCNCheck -SkipRevocationCheck
 
 #Options for a trusted SSL certificate
-$opt = New-CimSessionOption -UseSsl $true
-$Sess=New-CimSession -Credential $credential -ComputerName $Node -Port 5986 -Authentication basic -SessionOption $opt -OperationTimeoutSec 90
+$opt = New-CimSessionOption -UseSsl
+
+$Sess = New-CimSession `
```
Comment (chasewilson) [inline reference/docs-conceptual/dsc/getting-started/lnxGettingStarted.md:141]: Using the backticks this way are usually against best practice. Would you want to splat it or leave it long?

**Review by chasewilson — CHANGES_REQUESTED** [review]
Great to have you on here @dcrreynolds ;) Everything looks good except for one comment. Once we get that one taken care of, we'll get this merged in.

File: reference/docs-conceptual/dsc/getting-started/lnxGettingStarted.md (hunk 1 above)
Comment (doctordns) [reply reference/docs-conceptual/dsc/getting-started/lnxGettingStarted.md:141]: Please encourage splitting.

Comment (dcrreynolds) [reply reference/docs-conceptual/dsc/getting-started/lnxGettingStarted.md]: Added splatting and updated the ms.date!

**Review by chasewilson — APPROVED** [review]
Thanks @dcrreynolds !

**Pull request merged**